src/GraphQL/Language/Variables.cs

```diff
@@ -41,7 +42,7 @@ internal Variables(int initialCount)
         /// <summary>
         /// Gets the first variable with a matching name. Returns <see langword="true"/> if a match is found.
         /// </summary>
-        public bool ValueFor(string name, out ArgumentValue value)
+        public bool ValueFor(ROM name, out ArgumentValue value)
```

Comment:
@Shane32 You will see such a change in many places. Due to the implicit conversion of types, the code works as before:
```
ValueFor("abc");
ROM some = ....;
ValueFor(some);
GraphQLName name = ....;
ValueFor(name);
```